keras/callbacks.py

```diff
@@ -1253,6 +1253,9 @@ class ModelCheckpoint(Callback):
       options: Optional `tf.train.CheckpointOptions` object if
         `save_weights_only` is true or optional `tf.saved_model.SaveOptions`
         object if `save_weights_only` is false.
+      initial_best: Initial "best" value of the metric to be monitored.
```

Comment:
Let's rename this "initial_value_threshold" -- thanks!

Reply:
Please update the docstrings as well.

Reply:
@fchollet made the changes in docstring.